components/style/traversal.rs

```diff
@@ -229,15 +229,6 @@ fn ensure_node_styled_internal<'a, N, C>(node: N,
 {
     use properties::longhands::display::computed_value as display;
 
-    // Ensure we have style data available. This must be done externally because
-    // there's no way to initialize the style data from the style system
-    // (because in Servo it's coupled with the layout data too).
-    //
-    // Ideally we'd have an initialize_data() or something similar but just for
-    // style data.
-    debug_assert!(node.borrow_data().is_some(),
```

Comment:
I've added a comment. I didn't sweat it too much before because the invariants here are all going to change with the new architecture anyway.